src/sections/Learn-Layer5/Info/index.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import CardWrapper from "./info.style";
+
+function Info() {
+  return (
+    <CardWrapper>
+      <div className="row">
```

Comment:
You can import the `Row` and `Column` components from the `layout` file and use them here. Also, the `Container` component.
Component link: https://github.com/layer5io/layer5/blob/master/src/components/layout.js
For reference, you can check this [section](https://github.com/layer5io/layer5/blob/master/src/sections/Blog/Blog-grid/index.js) for usage.

Reply:
Just an update let's not use the `Container` in the component directly. Let it be dependent upon the section where it is used, so we get flexibility.

Reply:
You mean to enclose the `LearnInfo` component inside its caller, right?

Reply:
So you mean to add the `Container` component in the file where I have called `LearnInfo`?

Reply:
For the time being you can leave that. Just use the Row, and Column component in the LearnInfo component.

Reply:
Without using the `Container` in the LearnInfo the columns are not well organized. They are shifted towards the left. Does that make sense to you? 
![Screenshot 2021-05-22 at 1 22 32 AM](https://user-images.githubusercontent.com/53913514/119191396-30b89600-ba9c-11eb-9269-3c577b803b44.png)

Reply:
Check the comment below. It can be aligned by aligning the parent div.